src/com/xilinx/rapidwright/interchange/DeviceResourcesWriter.java

```diff
@@ -168,7 +169,9 @@ public static void populateEnumerations(Design design, Device device) {
             }
 
         }
-        for(Entry<String,Pair<String, EnumSet<IOStandard>>> e : EDIFNetlist.macroExpandExceptionMap.entrySet()) {
+        Map<String, Pair<String, EnumSet<IOStandard>>> macroExpandExceptionMap = 
+                EDIFNetlist.macroExpandExceptionMap.getOrDefault(device.getSeries(), Collections.emptyMap()); 
```

Comment:
```suggestion
        Map<String, Pair<String, EnumSet<IOStandard>>> macroExpandExceptionMap = 
        EDIFNetlist.macroExpandExceptionMap.getOrDefault(device.getSeries(), Collections.emptyMap()); 
```

Reply:
This indent is preferred to indicate a line continuation from the previous line.